src/App.jsx

```diff
@@ -1,104 +1,29 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+function findUser(userId) {
+  return usersFromServer.find(user => userId === user.id) || null;
+}
+
+function findComment(postId) {
+  return (
+    commentsFromServer.filter(comment => postId === comment.postId) || null
```

Comment:
The `findComment` function should return an empty array instead of `null` when no comments are found. This ensures consistency in the data structure, as the `comments` property should always be an array.